utils/config/config.go

```diff
@@ -651,3 +652,42 @@ func (missionControlDetails *MissionControlDetails) GetAccessToken() string {
 func (missionControlDetails *MissionControlDetails) SetAccessToken(accessToken string) {
 	missionControlDetails.AccessToken = accessToken
 }
+
+func CreateDefaultJfrogConfig() (err error, cleanUp func()) {
+	err, cleanUp = tests.SetJfrogHome()
+	if err != nil {
+		return
+	}
+	configuration := `
+		{
+  "servers": [
+    {
+      "artifactoryUrl": "http://localhost:8080/artifactory/",
+      "user": "user",
+      "password": "password",
+      "serverId": "name",
+      "isDefault": true
+    },
+    {
+      "artifactoryUrl": "http://localhost:8080/artifactory/",
+      "user": "user",
+      "password": "password",
+      "serverId": "notDefault"
+    }
+  ],
+  "version": "5"
+}
+	`
+	content, err := convertIfNeeded([]byte(configuration))
+	if err != nil {
+		return
+	}
+	configFilePath, err := getConfFilePath()
+	configFile, err := os.Create(configFilePath)
+	defer configFile.Close()
+	_, err = configFile.Write(content)
+	if err != nil {
```

Comment:
This if statement seem redundant, becuase the next line returns the error anyway.